controllers/components/codeflare/codeflare_controller_actions.go

```diff
@@ -9,15 +9,13 @@ import (
 	odhdeploy "github.com/opendatahub-io/opendatahub-operator/v2/pkg/deploy"
 )
 
-func initialize(ctx context.Context, rr *odhtypes.ReconciliationRequest) error {
```

Comment:
probably, "initialize" name for local actions which do applyParams only is too abstract.